src/constants.js

```diff
@@ -246,3 +246,7 @@ export const HTTP_REQUEST_TIMEOUT = 3000;
  * Any network that is not mainnet or testnet should be a privatenet.
  */
 export const NETWORK_PRIVATENET = 'privatenet';
+
+export const MAX_RETRIES_WS_CALL = 5;
```

Comment:
Why `WS_CALL`? Why not only `MAX_RETRIES`?

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-mobile/pull/444/commits/c552aa618f7da09b9353666901f6bfe4d6ca441e